conan/tools/ros/ament.py

```diff
@@ -0,0 +1,855 @@
+from conan.tools.files import save
+from conan.tools.cmake import CMakeDeps
+
+import os
+
+local_setup_bash = """\
```

Comment:
Where are these files coming from? 
It is a bit surprising having to embed all this stuff for this to integrate with ros

Reply:
I'd prefer not to include this by default if we don't really understand what they are doing and we can take ownership of them.

For example other ament package like this one https://github.com/ament/ament_cobra, doesn't seem to include these scripts. What are they doing? are they really necessary?

Reply:
These scripts are automatically generated when building a package with the `colcon` and `ament` tools.